src/main/resources/application-prod.yml

```
@@ -45,4 +45,8 @@ server:
     encoding:
       charset: UTF-8
       enabled: true
-      force: true
\ No newline at end of file
+      force: true
+
+
+jwt:
+  secret:${JWT_SECRET} # 반드시 환경변수 통해 시크릿 키 주입
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)